course/include/isometry/vector3.hpp

```diff
@@ -0,0 +1,152 @@
+/*
+ * Isometry library
+ * Copyright 2020 Ekumen Inc.
+ * Author: Alexis Pojomovsky, 2020
+ */
+
+#pragma once
+
+#include <cmath>
+#include <initializer_list>
+#include <iostream>
+#include <sstream>
+#include <string>
+
+namespace ekumen {
+
+namespace math {
+
+/**
+ * This class is used to represent a 3-dimensional vector of doubles.
+ */
+class Vector3 {
+ public:
+  /// \brief Default constructor.
+  Vector3();
+
+  /// \brief Constructor parametrized with 3 doubles.
+  Vector3(double _x, double _y, double _z);
```

Comment:
All three parameters can be `const`. Also, probably you did not meant to prefix them with `_` .